checks.d/docker_daemon.py

```diff
@@ -174,6 +174,7 @@ def init(self):
             self._disable_net_metrics = False
 
             # Set tagging options
+            self.include_app_id = instance.get("include_marathon_app_id", False)
```

Comment:
Nitpicking but can you rename this attribute to `include_marathon_app_id`? App is quite generic, it could be an issue when we add more tags in the future.